src/bundle/ui-dev/src/modules/universal-discovery/hooks/useSelectedLocationsHelpers.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useContext } from 'react';
+
+import { checkIsSelectable, checkIsSelected, checkIsSelectionBlocked } from '../helpers/selected.locations.helper';
+import {
+    AllowedContentTypesContext,
+    ContainersOnlyContext,
+    ContentTypesMapContext,
+    MultipleConfigContext,
+    SelectedLocationsContext,
+} from '../universal.discovery.module';
+
+export const useSelectedLocationsHelpers = () => {
+    const [, multipleItemsLimit] = useContext(MultipleConfigContext);
+    const contentTypesMap = useContext(ContentTypesMapContext);
+    const [selectedLocations] = useContext(SelectedLocationsContext);
+    const containersOnly = useContext(ContainersOnlyContext);
+    const allowedContentTypes = useContext(AllowedContentTypesContext);
+    const checkIsSelectableWrapped = useCallback(
```

Comment:
I have small issue with `Wrapped`, with current name it can be read as `check if IsSelectable wrapped`
maybe `cachedCheckIsSelectable`, or more complex
```
import { checkIsSelectable: checkIsSelectableHelper } from '../helpers/selected.locations.helper';

...

const checkIsSelectableWrapped = useCallback(
    (location) => checkIsSelectableHelper({ location, contentTypesMap, allowedContentTypes, containersOnly }),
    [contentTypesMap, allowedContentTypes, containersOnly],
);
```